mocks/internalpkg/microsoft/azure_open_ai_client.go

```diff
@@ -0,0 +1,96 @@
+// Code generated by mockery v2.42.1. DO NOT EDIT.
```

Comment:
should the spelling of `openai` be fixed in the filename here, to be consistent with the names of the plugins/provider? or is it automatic and expected?

Reply:
It is auto generated. It converts pascal case to snake case. We can rename the client to AzureOpenaiClient. Wdyt?

Reply:
@traut it is auto generated. Renaming the client to AzureOpenaiClient should fix it.

Reply:
@traut  it is auto generated. Renaming the client to AzureOpenaiClient should fix it.